### PR TITLE
List parser

### PR DESCRIPTION
**Pull request by ThunderMikey**
long waited :)

### CODE REVIEW

**Comment by ps-george** [comment]
Looks fine, but generating 

```
<ul>
  <li>ul1</li>
  <ul>
      <li>ul2</li>
      <ul><ul>
          <li>ul3</li>
      </ul></ul>
  </ul>
</ul>
```
instead of:
```
<ul>
  <li>ul1</li>
  <ul>
      <li>ul2</li>
      <ul>
          <li>ul3</li>
      </ul>
  </ul>
</ul>
```

**Comment by ps-george** [comment]
However, that doesn't affect the appearance of the html when it renders

**Comment by ThunderMikey** [comment]
@ps-george I set 2 whitespaces as 1 level of list depth. Maybe, you have a tab in front?(which might be 4 spaces)

**Comment by ps-george** [comment]
Ok, I will check that!

**Comment by ThunderMikey** [comment]
@ps-george  Maybe, 4 whitespaces is more common?

**Comment by ps-george** [comment]
I think maybe we need to have a difference between spaces and tabs in our lexer...

File: FMark/src/Common/Parser/Parser.fs [hunk 1]
```diff
@@ -112,12 +112,16 @@ let parseList toks =
                         xOnwards currentLine lines
                         |> getCurrentList (currentLv+1) []
                         |> parseList' (currentLv+1)
-                    (currentLv, NestedList(listItem)::listItems, skip, currentLine+1)
+                    if skip =  Some(1)
```
Comment (ThunderMikey) [inline FMark/src/Common/Parser/Parser.fs]: Does it make a difference? I think reverting back is more elegant.

Comment (ps-george) [reply FMark/src/Common/Parser/Parser.fs]: I reverted back and it caused some error so left it like this.

**Pull request merged**